src/isCompleted.js

```diff
@@ -0,0 +1,6 @@
+//  Module
+
+export default function returnsUncompleted(task) {
+  // console.log('In returnsUncompleted- task: ', task);
```

Comment:
- [ ] I would recommend removing all the commented code as removing commented code makes the codebase cleaner.